### PR TITLE
33 adding parity

### PR DESCRIPTION
**Pull request by geckoslair**
i've added new scripts:

"dev-parity": "concurrently \"yarn parity\" \"yarn dev-client\" \"yarn dev-server\"

and

"parity": "parity --config parity-config.toml"

paraty-config.toml and chain.json are used for rightly configure the dev chain.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "dev-client": "yarn clean && yarn prep && webpack --watch --colors --progress",
     "dev-server": "nodemon --watch src/server --watch node_modules src/server/index.js localhost:8080",
     "dev": "concurrently \"yarn dev-client\" \"yarn dev-server\"",
+    "dev-parity": "concurrently \"yarn parity\" \"yarn dev-client\" \"yarn dev-server\"",
```
Comment (bent0b0x) [inline package.json]: I would run `parity` concurrently in the `"dev"` script. We almost always will want to run parity when developing, yes?

**Comment by bent0b0x** [comment]
After @jellegerbrandy 's discussion, I think the target of this PR should be changed to `dev`. Additionally, in general this looks good to me. But I don't think we should merge it until the actual debug functionality has been added to the component. If we just want to merge the `parity` stuff, then I would remove the `/debug` route from this PR

**Comment by geckoslair** [comment]
rebased

File: package.json (hunk 1 above)
Comment (geckoslair) [reply package.json]: we had a configuration like this for testrpc on meteor, sometimes it's better run it separately for better debug. Change as you prefer man! =)

Comment (bent0b0x) [reply package.json]: Nah that makes sense, 👍 !

**Pull request merged**